src/js/breadcrumbs/breadcrumbs-view.js

```diff
@@ -1,17 +1,38 @@
+/**
+ * A Backbone view that renders breadcrumbs-type tiered navigation.
+ *
+ * Initialize the view by passing in the following attributes:
+ *
+ *~~~ javascript
+ * var view = new DropdownMenuView({
+ *     el: $('selector for element that will contain breadcrumbs'),
+ *     model: new BreadcrumbsModel({
+ *         breadcrumbs: [{url: '/', title: 'Overview'}]
+ *     }),
+ *     events: {
+ *         'click nav.breadcrumbs a.nav-item': function (event) {
+ *             event.preventDefault();
+ *             window.location = $(event.currentTarget).attr('href');
+ *         }
+ *     }
+ * });
+ *~~~
+ * @module BreadcrumbsView
+ */
 ;(function (define) {
     'use strict';
-    define(['backbone', 'underscore', 'text!./breadcrumbs.underscore'],
-        function (Backbone, _, breadcrumbsTemplate) {
+    define(['backbone', '../utils/html-utils.js', 'text!./breadcrumbs.underscore'],
```

Comment:
I had trouble consuming the UI Toolkit in edx-platform when using relative paths. I think you might need to use a full path like this:

https://github.com/edx/edx-ui-toolkit/blob/master/src/js/utils/html-utils.js#L14